src/core/CMakeLists.txt

```diff
@@ -151,6 +151,15 @@ if(${TRITON_ENABLE_S3})
   message(STATUS "Using aws-sdk-cpp ${AWSSDK_VERSION}")
 endif()
 
+if(${TRITON_ENABLE_GPU})
+  set(CUDA_LIBRARIES PUBLIC ${CUDA_LIBRARIES})
+  cuda_add_library(
+    gather_kernel 
```

Comment:
naming convention, use dash: "gather-kernel"

Reply:
I change the name to a more general name `kernel-library`, I can imagine we may try to expose the similar feature in output side (i.e. scatter kernel)